cmd/di_desktop.go

```diff
@@ -232,33 +235,25 @@ func (di *Dependencies) bootstrapServiceComponents(nodeOptions node.Options) err
 		log.Warn().Err(err).Msg("Failed to enable NAT forwarding")
 	}
 	di.ServiceRegistry = service.NewRegistry()
+
 	storage := session.NewEventBasedStorage(di.EventBus, session.NewStorageMemory())
+	if err := storage.Subscribe(); err != nil {
+		return errors.Wrap(err, "could not subscribe session to node events")
+	}
 	di.ServiceSessionStorage = storage
 
-	err := storage.Subscribe()
-	if err != nil {
-		return errors.Wrap(err, "could not bootstrap service components")
+	policyRepo := policy.NewPolicyRepository(di.HTTPClient, nodeOptions.AccessPolicyEndpointAddress, 10*time.Minute)
```

Comment:
Can we make this interval configurable? Could be used for e2e tests etc.

Reply:
Introduced